node/1.1.4-3/SASSDomain.js

```diff
@@ -199,26 +196,7 @@ function _nextRender() {
         cleanup();
 
         if (message.css) {
-            // Convert sources array paths to be relative to input file
-            var sourceMapPath = renderMsg._sourceMapPath,
-                mapJSON = JSON.parse(message.map),
-                sourcePath,
-                inputParent = path.dirname(renderMsg.file);
-
-            mapJSON.sources.forEach(function (source, index) {
-                // Resolve from working directory (e.g. c:\windows\system32)
-                sourcePath = path.resolve(message._cwd, source);
-                sourcePath = path.relative(inputParent, sourcePath);
-
-                if (path.sep === "\\") {
-                    sourcePath = sourcePath.replace(/\\/g, "/");
-                }
-
-                // Set source path relative to input file parent (sourceRoot)
-                mapJSON.sources[index] = sourcePath;
-            });
```

Comment:
(Lines 202-219)

I've moved this code into `render.js` because source maps generated by Ruby Sass are OK. By converting their source paths, paths like `includes/_partial.scss` become something like `../../../Windows/System32/includes/_partial.scss`